com.civicrm.hrjobroles/js/test/controllers/job-roles.controller.spec.js

```diff
@@ -50,10 +50,10 @@ define([
           };
         });
         initController();
-        ctrl.openOptionsEditor(url, optionType);
       });
 
       it('calls the crmAngService with the requested url', function () {
+        ctrl.openOptionsEditor(optionType);
```

Comment:
Why was it moved? We should keep `it` blocks as clean as possible.

Reply:
There is no specific reason. I just felt it should be there, showing action that took place and what is expected. Will be moving it back.